libs/packages/components/src/lib/collapse/collapse.component.scss

```diff
@@ -0,0 +1,3 @@
+.text-align-center {
```

Comment:
Do not create helper classes that exist in USWDS:

https://designsystem.digital.gov/utilities/paragraph-styles/